ansible-ipi-install/inventory/hosts.sample

```diff
@@ -88,6 +88,10 @@ prov_ip=172.22.0.3
 # (Optional) Increase install process timeout by N iterations.
 #increase_install_timeout=2
 
+# (Optional) Disable HW inspection to intelligently choose between IPMI or RedFish protocol.
+# By default this feature is enabled and set to true. Uncomment below to disable and use IPMI.
+# enable_hw_detection=false
```

Comment:
I find the variable name a bit confusing... can that be `redfish_inspection` instead? (true by default)

Reply:
Sure I can do that.

Reply:
@e-minguez changed. Can you confirm it works as expected on your env, and I will merge.